accounts-db/src/accounts_db/load_accounts_histogram.rs

```diff
@@ -0,0 +1,123 @@
+use {
+    hdrhistogram::Histogram,
+    std::time::{Duration, Instant},
+};
+
+#[derive(Debug)]
+pub struct LoadAccountsHistogram {
+    samples: Vec<Duration>,
+    previous_submit: Instant,
+}
+
+impl LoadAccountsHistogram {
+    const SUBMIT_INTERVAL: Duration = Duration::from_secs(60);
+
+    #[must_use]
+    pub fn new() -> Self {
+        Self {
+            samples: Vec::new(),
+            previous_submit: Instant::now(),
+        }
+    }
+
+    /// Records `sample`
+    ///
+    /// Will be included in the next submission
+    pub fn record(&mut self, sample: Duration) {
+        self.samples.push(sample);
```

Comment:
We add a sample everytime we load an account. Should we put a bound on number of samples so this doesn't grow too large?
I think reasonably it wouldn't, but potentially, although probably very unlikely, there's a resource exhaustion attack here with a spammer sending txs loading many accounts that fail quickly so banking stage is able to process a lot of them?